components/text/aspell/Makefile

```diff
@@ -19,47 +19,42 @@
 # CDDL HEADER END
 #
 # Copyright 2011 EveryCity Ltd. All rights reserved.
+# Copyright 2019 Michal Nowak
 #
+
+BUILD_BITS=		32_and_64
+
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		aspell
-COMPONENT_VERSION=	0.60.6.1
-COMPONENT_REVISION=	2
+COMPONENT_VERSION=	0.60.8
 COMPONENT_FMRI=		text/aspell
 COMPONENT_SUMMARY=	GNU Aspell spell checking library
 COMPONENT_CLASSIFICATION=System/Text Tools
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	sha256:f52583a83a63633701c5f71db3dc40aab87b7f76b29723aeb27941eff42df6e1
+COMPONENT_ARCHIVE_HASH=	sha256:f9b77e515334a751b2e60daab5db23499e26c9209f5e7b7443b05235ad0226f2
 COMPONENT_ARCHIVE_URL=	http://ftp.gnu.org/gnu/$(COMPONENT_NAME)/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL=	http://aspell.net/	
 COMPONENT_LICENSE=	LGPLv2.1
 
+TEST_TARGET=		$(NO_TESTS)
 
-include $(WS_TOP)/make-rules/prep.mk
-include $(WS_TOP)/make-rules/configure.mk
-include $(WS_TOP)/make-rules/ips.mk
+include $(WS_MAKE_RULES)/common.mk
 
 CXXFLAGS+=-I/usr/include/ncurses
 
 LDFLAGS.32=	$(LDFLAGS) -L$(GNULIB) -R $(GNULIB)
 LDFLAGS.64=	$(LDFLAGS) -L$(GNULIB64) -R $(GNULIB64)
 
-COMPONENT_PRE_CONFIGURE_ACTION += cp -r $(SOURCE_DIR)/* $(@D) && \
-			cd $(@D) && autoconf
 CONFIGURE_OPTIONS+=	--sysconfdir=/etc 
 CONFIGURE_OPTIONS+=	--enable-shared
 CONFIGURE_OPTIONS+=	--disable-static 
 CONFIGURE_OPTIONS+=	--with-pic 
-CONFIGURE_OPTIONS+=	--with-curses=ncurses 
```

Comment:
Does it select ncurses by default now?

Reply:
Changed that to `--enable-curses="-lncurses"`:
```
  --enable-curses=LIBFILE cursor control library
  --enable-curses-include=DIR
  --disable-wide-curses   disable wide char utf8 cursor control
```
Appears to work fine:
```
 newman  ~  for bin in `pkg contents aspell | grep bin`; do ldd /$bin 2>/dev/null | grep curses && echo "* $bin"; done
	libncurses.so.5 =>	 /usr/lib/64/libncurses.so.5
* usr/bin/amd64/aspell
	libncurses.so.5 =>	 /usr/lib/libncurses.so.5
* usr/bin/aspell
```